### PR TITLE
Added Hadrian's Wall Game

### PR DESCRIPTION
**Pull request by ShilpiBG12**
Hi,
I have added this new game "Hadrian's Wall". Please see if you merge this contribution against Hacktoberfest. As just 1 more PR need to be completed for completing 4 contributions in Hacktoberfest event, please see if you can merge it fast. Thank you so much for considering. It would be a great help.

Submitting this PR means you have:

- [x] reviewed the [contributing][contributing-url] document and are comfortable understanding what it says
- [x] searched the [readme][readme-url] file and searched the [open pull requests][pr-url] (no duplicates please)
- [x] and will follow [the desired format][format-url]
- [x] used image links that resolve correctly, are appropriately sized, and are served over HTTPS
- [x] read [this year's Hacktoberfest welcome issue](https://github.com/edm00se/awesome-board-games/issues/269)
- [ ] promised not to `@` me asking for review (I will see the notifications already, no need to add noise)

Once your PR is created, it will trigger a build a couple of automated checks, including a GitHub Action (Awesome-Lint). This includes a format linting process, so if there are any issues with the build, please address them. The others include a check on the submission limit and also a preview of the site with your changes; please check them out, they are tools tonhelp us both.

Do you have further questions? Check the [contribution guidelines][contributing-url] or [open an issue][new-issue-url].

When you're ready, check off the above acknowledgements and continue with your Pull Request... and thanks!

[contributing-url]: https://github.com/edm00se/awesome-board-games/blob/main/contributing.md
[pr-url]: https://github.com/edm00se/awesome-board-games/pulls
[readme-url]: https://github.com/edm00se/awesome-board-games/edit/main/readme.md
[format-url]: https://github.com/edm00se/awesome-board-games/blob/main/formatting.md
[contributing-url]: https://github.com/edm00se/awesome-board-games/blob/main/contributing.md
[new-issue-url]: https://github.com/edm00se/awesome-board-games/issues/new

### CODE REVIEW

**Comment by ShilpiBG12** [comment]
@edm00se Sorry to bother you at a short notice. But please see if you can approve this PR under Hacktoberfest. Thanks a lot.

File: readme.md [hunk 1]
```diff
@@ -2881,17 +2881,16 @@ Superfight is a game where you argue with your friends over ridiculous fights. T
...
+>
+> In Hadrian's Wall, players take on the role of a Roman General placed in charge of the construction of a milecastle and bordering wall. Over six years (rounds), players will construct their fort and wall, man the defenses, and attract civilians by building services and providing entertainment — all while defending the honor of the Roman Empire from the warring Picts. The player who can accumulate the most renown, piety, valor and discipline, whilst avoiding disdain, will prove to the Emperor they are the model Roman citizen and be crowned Legatus Legionis!
 
-### [Twilight Inscription](https://boardgamegeek.com/boardgame/361545/twilight-inscription)
```
Comment (edm00se) [inline readme.md:2885]: Do not remove existing entries.

**Review by edm00se — CHANGES_REQUESTED** [review]
The addition looks fine but this Pull Request will not be accepted as is since your current changes are replacing another entry. Also please re-read the last checkbox provided by the PR template for the description; you checked it, but still `@` mentioned me. I will review this PR again later on after you've made your updates to fix the removal of the other game.

**Comment by edm00se** [comment]
Closing this one as it's now a duplicate.

File: readme.md (hunk 1 above)
Comment (ShilpiBG12) [reply readme.md:2885]: Sorry for this. Will take proper care in future.

**Pull request closed without merging**